sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/KeyVaultClient.java

```diff
@@ -167,249 +142,249 @@ public KeyVaultClient(String keyVaultUri, String loginUri, String tenantId, Stri
         this.keyVaultUri = keyVaultUri;
         // Base Uri shouldn't end with a slash.
         String domainNameSuffix = Optional.of(keyVaultUri)
-                                          .map(uri -> uri.split("\\.", 2)[1])
-                                          .map(suffix -> suffix.substring(0, suffix.length() - 1))
-                                          .orElse(null);
-        this.keyVaultBaseUri = validateUri(HTTPS_PREFIX + domainNameSuffix, "Key Vault URI");
-        this.aadAuthenticationUri = addTrailingSlashIfRequired(
-            loginUri != null
-                ? validateUri(loginUri, "Login URI") // Validate any user-provided login URI.
-                : getAADLoginURIByKeyVaultBaseUri(keyVaultBaseUri)); // These are all valid URIs.
-
+                .map(uri -> uri.split("\\.", 2)[1])
+                .map(suffix -> suffix.substring(0, suffix.length() - 1))
+                .orElse(null);
+        keyVaultBaseUri = HTTPS_PREFIX + domainNameSuffix;
```

Comment:
Any reason for removing URI validation?

Reply:
I just reverted the commit that added beta functionality we're moving away from. I will add back the validation as it is a good thing to keep around.